### PR TITLE
Integrate search functionality with freshdesk

### PR DESCRIPTION
**Pull request by aurimas-resdiary**

### CODE REVIEW

File: commands/SearchCommand.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const commandName = 'search';
+require('dotenv').config();
+const Freshdesk = require('../helpers/freshdesk');
+const freshdesk = new Freshdesk(process.env.FRESHDESK_URL, process.env.FRESHDESK_API_KEY);
```
Comment (Scott-Resdiary) [inline commands/SearchCommand.js]: This constants can be moved into the FreshDesk constructor and removed from this file

File: commands/SearchCommand.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+const slackBlocks = require('../helpers/slack_blocks');
+const SlackSearchBlock = slackBlocks.SlackSearchBlock;
+
+
+const searchCommand = function(controller) {
+    
+    controller.hears('search', ['direct_message','direct_mention'], async(bot, message) => {
+        const msg = message.text;
+        const searchTerm = msg.replace('search', '').trim();
```
Comment (Scott-Resdiary) [inline commands/SearchCommand.js]: does the leading space impact this search?

File: commands/SearchCommand.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+        await bot.reply(message,{
+            blocks: SlackSearchBlock(searchTerm, solutions)
+        });
+
+    });
+
+}
+
+searchCommand.description = `\`${commandName}\` - searches articles from several sources`;
```
Comment (Scott-Resdiary) [inline commands/SearchCommand.js]: Need to discuss this, Might just be updating description

File: helpers/utils.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    
+    let result = await axios.get(url, { headers: config }).then(response => {
+            // If request is good...
+            return response.data;
+        })
+        .catch((error) => {
+            console.log('error 3 ' + error);
+        });
+
+    return result;
```
Comment (Scott-Resdiary) [inline helpers/utils.js]: Could maybe move this to a generic Axios helper class? Might no tbe worth it

File: commands/SearchCommand.js (hunk 2 above)
Comment (aurimas-resdiary) [reply commands/SearchCommand.js]: doesn't seem so, but might better to trip anyway?

**Pull request merged**